dist-test/index.js

```diff
@@ -56,21 +56,13 @@ function isGlamorousFunction(thing) {
     )
     return false
   }
-  if (thing.name !== 'glamorous') {
```

Comment:
I found that the name doesn't really matter all that much and the UMD/CJS builds have glamorous with a different name. NBD :)